neptune-core/src/application/json_rpc/core/model/block/transaction_kernel.rs

```diff
@@ -109,3 +155,24 @@ impl From<&TransactionKernel> for RpcTransactionKernel {
         }
     }
 }
+
+impl From<RpcTransactionKernel> for TransactionKernel {
+    fn from(kernel: RpcTransactionKernel) -> Self {
+        TransactionKernelProxy::from(kernel).into_kernel()
+    }
+}
+
+impl From<RpcTransactionKernel> for TransactionKernelProxy {
```

Comment:
Is this conversion method needed? If you're not using it, please delete. As far as I can see this doesn't make a lot of sense since `TransactionKernelProxy` is an interface for changing values in an already-existing transaction kernel.